test/performance/workloads/cluster-density-ms/bindingPolicy.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: control.kubestellar.io/v1alpha1
+kind: BindingPolicy
+metadata:
+  name: {{.Name}}
+  labels:
+    kubestellar: performance-test
+spec:
+  clusterSelectors:
+  - matchLabels: {"location-group":"edge"}
+  downsync:
+  - objectSelectors:
+    - matchLabels: {"app.kubernetes.io/name":{{.app}}-{{.Index}} }
```

Comment:
The labels in the API object YAMLs below use a literal `cluster-density` instead of `{{.app}}`. Why the difference?

Reply:
The syntax  ``{"app.kubernetes.io/name":index-{{.Index}} }`` was raising parsing error. Hence, I replaced it with the current syntax: ``{"app.kubernetes.io/name":{{.app}}-{{.Index}} }``

Reply:
The syntax ``{"app.kubernetes.io/name":cluster-density-{{.Index}} }`` was raising parsing error. Hence, I replaced it with the current syntax: ``{"app.kubernetes.io/name":{{.app}}-{{.Index}} }``

Reply:
Did you try putting dquotes around that label value?
There is not a critical problem here, I am just curious.

Reply:
That works! Thanks for this suggestion : )